libvast/test/event.cpp

```diff
@@ -45,7 +45,7 @@ struct fixture {
   }
 
   type t;
-  vector r;
+  std::vector<data> r;
```

Comment:
```suggestion
  list r;
```

Reply:
See my comment below for why going with `std::vector<data>`.